ceph/rados/pool_workflows.py

```diff
@@ -740,7 +740,7 @@ def modify_autoscale_threshold(self, threshold: float):
         Returns: None
         """
         cmd = f"ceph osd pool set threshold {threshold}"
-        self.rados_obj.run_ceph_command(cmd=cmd)
+        self.rados_obj.node.shell([cmd])
```

Comment:
self.node is also initiated in this class, can we use it directly?

Reply:
Changed the call.